### PR TITLE
interface methods: Speedy

### PR DESCRIPTION
**Pull request by sofiafaro-da**
Part of #11006

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Good stuff, thanks for getting this ready so quickly!

File: daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala [hunk 1]
```diff
@@ -1144,8 +1144,12 @@ private[archive] class DecodeV1(minor: LV.Minor) {
           )
 
         case PLF.Expr.SumCase.CALL_INTERFACE =>
-          // TODO https://github.com/digital-asset/daml/issues/10810
-          throw Error.Parsing("Expr.call_interface not yet implemented")
+          val callInterface = lfExpr.getCallInterface
```
Comment (remyhaemmerle-da) [inline daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala:1147]:
```suggestion
          assertSince(LV.Features.interfaces, "Expr.call_interface")
          val callInterface = lfExpr.getCallInterface
```

**Review by remyhaemmerle-da — APPROVED** [review]
Nice.

File: daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala (hunk 1 above)
Comment (sofiafaro-da) [reply daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala:1147]: Good idea, thank you!

**Pull request merged**